charts/sysdig/values.yaml

```
@@ -479,6 +487,12 @@ nodeAnalyzer:
       digest:
       pullPolicy: IfNotPresent
 
+    extraMounts: []
+    # example for bottlerocket
+    # extraMounts: []
```

```suggestion
    # extraMounts:
```
this is a typo I think